spec/factories/events.rb

```diff
@@ -123,6 +123,12 @@
       end
     end
 
+    trait :with_enough_free_places do
+      after(:build) do |event|
+        event.max_participants = event.application_letters.size + 1
```

Comment:
Why do you need the addition plus one here? Or is it just for safety reason if there are no application letters or something else?